lib/Remotery.c

```diff
@@ -5205,51 +5206,32 @@ RMT_API void _rmt_LogText(rmtPStr text)
     start_offset = 8;
 
     // There might be newlines in the buffer, so split them into multiple network calls
-    prev_offset = start_offset;
+    offset = start_offset;
     for (i = 0; text[i] != 0; i++)
     {
         char c = text[i];
 
         // Line wrap when too long or newline encountered
-        if (prev_offset == sizeof(line_buffer) - 3 || c == '\n')
+        if (offset == sizeof(line_buffer) || c == '\n')
```

Comment:
There used to be an extra -1 here so that `line_buffer` could be viewed in the debugger using the null termination above (`line_buffer = { 0 }`). Can you restore that?

I've also noticed that setting `start_offset = 8` leaves zeroes in-between so you can no longer view the string in a debugger without offsetting it but that's my bug!

Reply:
Ah, I see! Makes sense, I restored it and also added the spaces.